components/raftstore-v2/tests/integrations/test_pd_heartbeat.rs

```diff
@@ -59,3 +66,58 @@ fn test_store_heartbeat() {
     }
     panic!("failed to get store stats");
 }
+
+#[test]
+fn test_report_buckets() {
+    let region_id = 2;
+    let cluster = Cluster::with_node_count(1, None);
+    let store_id = cluster.node(0).id();
+    let router = &cluster.routers[0];
+
+    // When there is only one peer, it should campaign immediately.
+    let mut req = RaftCmdRequest::default();
+    req.mut_header().set_peer(new_peer(store_id, 3));
+    req.mut_status_request()
+        .set_cmd_type(StatusCmdType::RegionLeader);
+    let res = router.query(region_id, req.clone()).unwrap();
+    let status_resp = res.response().unwrap().get_status_response();
+    assert_eq!(
+        *status_resp.get_region_leader().get_leader(),
+        new_peer(store_id, 3)
+    );
+    router.wait_applied_to_current_term(region_id, Duration::from_secs(3));
+
+    // load data to split bucket.
+    let header = Box::new(router.new_request_for(region_id).take_header());
+    let mut suffix = String::from("");
+    for _ in 0..200 {
+        suffix.push_str("fake ");
+    }
+    for i in 0..10000 {
```

Comment:
Why so many keys? Is it possible to tune the config instead?

Reply:
fixed